src/classes/astyle.cpp

```diff
@@ -5,8 +5,13 @@
 #include "src/testing/adebug.h"
 #include "src/classes/asettings.h"
 
+#ifdef __linux__
```

Comment:
We should look into how fusion vs native looks on windows, maybe fusion might be a suitable default on all platforms?